docs-chef-io/content/server/upgrades.md

```diff
@@ -37,13 +37,47 @@ Requires License
 Supported Release
 : Chef Infra Server 13 and later are currently supported Chef Software releases. Earlier releases are no longer supported as of 12/31/2020. For more information about supported Chef Software see the [Supported Versions]({{< relref "/versions#supported-commercial-distributions" >}}) documentation.
 
+## Release-Specific Steps
+
 ### Upgrading to 14.x
 
-Chef Infra Server 14 moved from Solr to Elasticsearch as its search index.
+Chef Infra Server 14.0 moved from Solr to Elasticsearch as its search index.
 {{% server_upgrade_duration %}}
 
 The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.
 
+#### Upgrading to 14.8
+
+Chef Infra Server 14.8 upgraded PostgreSQL from 9.6 to 13.3. The 14.8 upgrade process requires a one time downtime for vacuuming, upgrading, and re-indexing the database. The entire upgrade operation takes approximately 1 minute per 1000 nodes. This process may take longer depending on your server hardware and the size of the node objects in your Chef Infra Server.
+
+##### Database Preparation
+
+1. We recommend running `VACUUM FULL` on the PostgreSQL database if you don't have automatic vacuuming set up. This process will reduce the size of the database by deleting unnecessary data and speeds up the migration. The `VACUUM FULL` operation takes around 1 to 2 minutes per gigabyte of data depending on the complexity of the data. Additionally, the `VACUUM FULL` operation requires additional free disk space at least as large as the size of your database.
+
+    ```bash
+    /opt/opscode/embedded/bin/vacuumdb --all --full
```

Comment:
It might be helpful to mention that these are the paths for standalone and tiered setup not external.